golos.publication/golos.publication.cpp

```diff
@@ -851,17 +848,18 @@ bool publication::check_permlink_correctness(std::string permlink) {
     return true;
 }
 
-void publication::check_curators_prcnt(uint16_t curators_prcnt) {
+base_t publication::check_curators_prcnt(int64_t curators_prcnt) {
```

Comment:
It’s unexpected for a programmer that this function returns a result.
It is also unexpected for the user that the minimum percentage will be set instead of the specified percentage. In this case, it's need to throw an exception that the value does not match the parameters.
`uint16_t` type is sufficient to store the value.